rotkehlchen/db/upgrades/v43_v44.py

```diff
@@ -122,6 +122,12 @@ def _add_uniswap_to_historical_oracles(cursor: 'DBCursor') -> None:
     )
 
 
+@enter_exit_debug_log()
+def _add_exited_timestamp(cursor: 'DBCursor') -> None:
+    """Add exited_timestamp column to the eth2_validators table"""
+    cursor.execute('ALTER TABLE eth2_validators ADD COLUMN exited_timestamp INTEGER')
```

Comment:
cuestion! Does sqlite assing NULL by default to an `INTEGER` nullable column? I would expect so but I'm not sure if it does something weid like giving a default value

Reply:
it's NULL yes